care/emr/resources/user/spec.py

```
@@ -101,7 +102,7 @@ def validate_user_email(cls, email):
             raise ValueError("Email already exists")
         try:
             validate_email(email)
-        except ValidationError as e:
+        except ValueError as e:
             raise ValueError("Invalid Email") from e
```

Ruff (0.14.7)

Avoid specifying long messages outside the exception class

(TRY003)